doc: Expand section on Getting Started (#19072)
bitcoin/bitcoin: .github/ISSUE_TEMPLATE/good_first_issue.md

---

> <!-- A good first issue is an uncontroversial issue, that has a more or less unique and obvious solution -->

Reviewer: ```suggestion
<!-- A good first issue is an uncontroversial issue, that has a relatively unique and obvious solution -->
```

<!-- Motivate the issue and explain the solution briefly -->

#### Useful skills:

<!-- (For example, “C++11 std::thread”, “Qt5 GUI and async GUI design” or “basic understanding of Bitcoin mining and the Bitcoin Core RPC interface”.) -->

#### Want to work on this issue?

> The purpose of the `good first issue` label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.

Reviewer 2: I'm ~0 on removing this. You might want to replace it with a reference to the CONTRIBUTING.md guide at least.

Reviewer 3: There should already be a link to CONTRIB.md
